sparql/updates/generate-top-level-mappings.rq

```diff
@@ -0,0 +1,26 @@
+PREFIX fhir: <http://hl7.org/fhir/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+
+# For every ICD-10 code in this file, find a "top-level code" (e.g. ICD-10 code "R50.9" has a top-level code of "R50")
+# and convert that into a concept IRI. Rather than adding that directly to the coding as we did previously, we want to
+# regenerate the concept IRI (i.e. for `R50.9`) and then generate a triple to say that ICD-10:R50.9 skos:broader ICD-10:R50.
+
+INSERT {
+    GRAPH <https://fhircat.org/reasoning1/top-level-icd10-mappings> {
+        ?concept_iri_code skos:broader ?concept_iri_top_level
+    }
+}
+# SELECT ?concept_iri_code ?concept_iri_top_level
+WHERE {
+    { SELECT DISTINCT ?concept_iri_code ?concept_iri_top_level WHERE {
```

Comment:
I don't think you need a subquery here.

Reply:
This is intended to reduce the number of INSERTs on the assumption that Blazegraph isn't smart enough to realize that it is going to see a LOT of repeated `?concept_iri_code`s from all the different conditions. But if Blazegraph is smart enough to do that, then yes, I can remove this subquery!

Reply:
To be honest, I'm not sure.